integrationTests/singleShard/block/interceptedRequestHdr_test.go

```diff
@@ -49,8 +49,8 @@ func TestNode_GenerateSendInterceptHeaderByNonceWithNetMessenger(t *testing.T) {
 		shardCoordinator,
 	)
 
-	nRequestor.Start()
-	nResolver.Start()
+	_ = nRequestor.Start()
+	_ = nResolver.Start()
 	defer func() {
 		_ = nRequestor.Stop()
```

Comment:
requestor?

Reply:
Removed all occurrences from the project